Libs/Image/Image.cpp

```diff
@@ -391,4 +393,82 @@ bool Image::pad(int padding, PixelType value)
 #endif
   return true;
 }
+
+bool Image::warp(std::string &source_file, std::string &target_file, int pointFactor)
```

Comment:
This function can just live in ImageUtils, produce the Transform, and call Image::applyTransform. 
This function has been in place for some time, but not yet in master.